### DISCUSSION TITLE
blog/next-js-blog-codeblock-styling

### DISCUSSION THREAD

**Discussion by giscus** [Likilee/kilee.dev#15] [Comments]
# blog/next-js-blog-codeblock-styling

이 글은 Next.js 블로그에서 rehype-pretty-code 플러그인을 사용하여 코드 블럭에 제목, 라인 넘버, 줄 단위 및 단어 단위 하이라이팅을 추가하는 방법을 다룹니다. 이 플러그인을 사용하여 블로그 글을 더 깔끔하고 이해하기 쉽게 작성할 수 있습니다.

https://kilee.dev/blog/next-js-blog-codeblock-styling

**Comment by jiwoniverse** [comment]
nextjs로 블로그 제작 중인데, 코드블록 스타일링에 도움이 많이 됐습니다!! 좋은 글 감사합니다 ㅎㅎ

**Reply by Likilee** [reply]
많이 부족한 글인데 도움이 되셨다니 다행입니다. 
응원 감사합니다 :)
